src/overlay/FlowControl.h

```diff
@@ -160,10 +172,7 @@ class FlowControl
 
     Json::Value getFlowControlJsonInfo(bool compact) const;
 
-    void
-    start(NodeID const& peerID,
-          std::function<void(std::shared_ptr<StellarMessage const>)> sendCb,
-          std::optional<uint32_t> enableFCBytes);
+    void start(NodeID const& peerID, std::optional<uint32_t> enableFCBytes);
```

Comment:
If we don't have sendCb anymore, is it still necessary to have a start method separate from the constructor?

Reply:
As of now, yes: we still need to set mNodeID and mFlowControlBytesCapacity based on what we get during auth. That being said, https://github.com/stellar/stellar-core/issues/3781 will allow to simplify the flow and we'll be able to initialize mFlowControlBytesCapacity in the constructor. We'll still need to set mNodeID after auth, but that's minor, since we only use peer name for logging.